src/routes/search/searchHelpers.ts

```diff
@@ -0,0 +1,23 @@
+// import type { Hit } from "@algolia/client-search";
+// import type { AlgoliaHit, HitHighlightResult } from "instantsearch.js";
```

Comment:
I'm still having some TypeScript woes on this page trying to define the expected shape of the search hits